templates/docker-compose-fragment.yml.j2

```diff
@@ -34,8 +39,13 @@ services:
   elasticsearch2:
     hostname: elasticsearch2
     environment:
+      {% if image_flavor == 'platinum' -%}
       - ELASTIC_PASSWORD=pleasechangeme
       - ALLOW_INSECURE_DEFAULT_TLS_CERT=true
+      {% endif -%}
+      {% if 'oss' not in image_flavor -%}
```

Comment:
and in 97e034012d5a36b824d93943e43afbd99da36ddc and a9bf1a67069333dc5276e94964f62d7ab975da47.